### ISSUE TITLE
Fix nyccamp_schedule view

### ISSUE THREAD

**Issue by esod** [NYC-Camp/website#88]
This view, admin/structure/views/view/nyccamp_schedule/edit contains all kinds of things we want to view, but it's broken.

You can see that it is the exact same view as last year:

http://2013.nyccamp.org/admin/structure/views/view/nyccamp_schedule/edit

Last year's view is not broken and you can see what we're viewing on.

There are eight displays in the view. You can delete/ignore the Your schedule display. We've built a whole module for that concept. We may want to think about this view as two views, or maybe not. First we need to fix the bug.

**Comment by jmarkel** [comment]
I've been looking at this a little. No fix as of yet, but as you may or may not have noticed, when you open the view for 2014 you get this message
![2014_nyccamp_schedule_message](https://f.cloud.github.com/assets/3930383/2317768/0bda51b8-a355-11e3-9192-5f3763d90fdc.jpg)

On the 2013 site, the Master display gives you:
![2013_nyccamp_schedule content nyc camp](https://f.cloud.github.com/assets/3930383/2317773/33de4548-a355-11e3-990b-250d02575f80.jpg)

while the 2014 Master display for this view shows a broken/missing field collection handler:
![2014_nyccamp_schedule content nyc camp-1](https://f.cloud.github.com/assets/3930383/2317801/168ac2d6-a356-11e3-83da-92c27548c20d.jpg)

@esod

**Comment by jmarkel** [comment]
@esod I doubt it's actually a missing handler - possibly the entity ref or one of the content types has changed slightly between COD versions. In any case I haven't yet looked further - will try to get at it today but lots going on :-/

**Comment by ForestMars** [comment]
@esod Did you really just publish our site root password on a public Github page? 

I just edited your post to remove it, but please do try to think before doing things like that.

**Comment by esod** [comment]
You're right. My bad. I'm sorry. Will you change that right away please?

## 

Eric Sod

**Comment by esod** [comment]
Thanks. I'm still not wholly used to working in the public space. 

We should also change the password on 2103. It might be inconvenient but it will secure the situation. 

## 

Eric Sod

**Comment by esod** [comment]
I changed 2013's password and now I can't log in. Still trying to get the hang of LastPass. Whoops. 

I assume @ForestMars you are NYC Camp and you should receive the email with the one time login.

Apologies for the confusion. At least the site is secure this way.

**Comment by mirie** [comment]
I'll take care of the password reset -- ping me in #nyccamp please.

**Comment by esod** [comment]
Okay. I got it working.

**Comment by esod** [comment]
The solution was not trivial.

Session Time Slot(s) field was changed this year. Session time Slot(s) is a concatenation of room assignments and timeslots. I'm finding it a very difficult field to work with.

So I made a new field called Session Timeslot Session (not the best name, maybe) and could make the relationships in the view work on that field because that field contains only the timeslot data.

We already have a Room assignment field which is our custom field and which works very well in a number of places.

Session Timeslot Session field is essentially the same as the Session Time Slot(s) field was last year. 

So then the pieces fell together.

I don't think we'll have a need for the concatenated Session Time Slot(s) field, and oob the field is hidden. Making Sesion time Slot(s) embedded was one of the early modifications I applied to COD this year. https://drupal.org/node/2145525. So we can change the concatenated field back to hidden. That way the admin won't be confusing with a Session Time Slot(s) field and a Session Timeslot Session (gotta come up with a better name) field.
